ansible/playbook/tasks/update.yml

```diff
@@ -168,6 +231,20 @@
       command: /usr/share/percona-dashboards/fix-panels.py
       changed_when: False
 
+    - name: Check if Grafana Image Renderer plugin has already installed
+      stat: path=/var/lib/grafana/plugins/grafana-image-renderer
+      register: gird
```

Comment:
May we rename it? Looks like a typo of "grid" :)

Reply:
g - grafana
r - renderer
i - image
d - directory

Have you got any proposition?

Reply:
`image_renderer_path`